README.md

```diff
@@ -64,3 +87,8 @@ You will need to do this if we need to add a new secret variable (API key, etc.)
 6. Run `travis encrypt myKey=mySecretValue --add env.global`. This will encrypt `mySecretValue` and add it to the `.travis.yml` file
 7. Add a comment in the `.travis.yml` file mapping `myKey` and the first 5 characters of the ciphertext
 8. Access your new secret variable in the code by calling `process.env.myKey`. Note that this value will only be populated on Travis-CI build servers
+
+##Testing Considerations
+
+*To be completed*
+- Remember to install Karma-cli globally
```

Comment:
I'm not sure this is true.. I was reading up on `npm run` and I think it spawns a shell with all of your `npm_modules` in the `PATH` - which would explain why it works on travis